.github/issue_label_bot.yaml

```diff
@@ -0,0 +1,4 @@
+label-alias:
```

Comment:
could you add a comment at the top of the file linking to mlbot.net and briefly explaining what this file is for.

Reply:
Ok done